sourcegraph/sourcegraph.proto

```diff
@@ -665,11 +668,22 @@ message TagList {
 	ListResponse list_response = 2 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
 }
 
+message MirrorReposRefreshVCSOp {
+	RepoSpec repo = 1 [(gogoproto.nullable) = false];
+	VCSCredentials credentials = 2;
+}
+
+// VCSCredentials for authentication during communication with VCS remotes.
+message VCSCredentials {
+	// Pass is the password provided to the VCS.
+	string pass = 1 [(gogoproto.nullable) = false];
```

Comment:
I haven't seen the related change in the MirrorRepos.RefreshVCS implementation, so I may not have full context, but it is peculiar that there is no login field in the VCSCredentials message. Is the login implicit in the operation somehow? Would it be cleaner to have it explicit here?